src/toast/_libtoast_pixels.cpp

```diff
@@ -60,6 +60,6 @@ void init_pixels(py::module & m) {
             (tuple):  The (local submap, pixel within submap) for each global pixel.
 
     )");
-    m.def("global_to_local", &global_to_local <int32_t>);
-    m.def("global_to_local", &global_to_local <int16_t>);
+    m.def("global_to_local", &global_to_local <int32_t> );
```

Comment:
Looks like `uncrustify` has slightly changed.

Reply:
Yeah, I am using version 0.69 from ubuntu sources.  I see upstream is at version 0.71.  If we become too annoyed in the future we could enforce a particular version...